### PR TITLE
Add variations of Company Cards

### PR DESCRIPTION
**Pull request by Jexeones24**
[CH 14228](https://app.clubhouse.io/policygenius/story/14228/engineers-can-utilize-a-companycard-component)
Company Cards will always have a logo or image. Company Cards may
optionally have a star rating (1-5) and/or a link. Star ratings and link
text and url are are passed in as props.

### CODE REVIEW

File: todo.md [hunk 1]
```diff
@@ -11,3 +11,4 @@
 - [ ] **TODO: put transition back** : ./src/static/MainNavNew/Menu/menu.module.scss
 - [ ] **TODO: put transition back** : ./src/static/MainNavNew/Search/search.module.scss
 - [ ] **TODO: put transition back** : ./src/static/MainNavNew/Search/search.module.scss
+
```
Comment (PepperTeasdale) [inline todo.md]: I think you'll want to remove this file from your commit.

File: src/organisms/cards/CompanyCard/company_card.module.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  img {
+    max-width: 100%;
+    max-height: 100%;
+    display: block;
+  }
+}
+
+.small {
+  min-height: rem-calc(172px);
```
Comment (PepperTeasdale) [inline src/organisms/cards/CompanyCard/company_card.module.scss]: Nitpick, but you don't need to add `px` when using `rem-calc`

File: src/organisms/cards/CompanyCard/company_card.module.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  min-width: rem-calc(172px);
+
+  .image-wrapper {
+    height: rem-calc(41px);
+    max-width: rem-calc(124px);
+    max-height: rem-calc(41px);
```
Comment (sunnymis) [inline src/organisms/cards/CompanyCard/company_card.module.scss]: We have a set of spacer guidelines we use, which can be found here https://app.zeplin.io/project/59cd15548a18a8bb0bc48b4d/dashboard?tag=Spacer. 

Zeplin is usually a few pixels off use the closest spacer.

File: src/molecules/StarRating/Readme.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+### StarRating Example:
+
+```jsx
+    <StarRating>
+      <strong>New Component Created!</strong><br /><small>Now go and write some code! -- <em>You code monkey, you ;)</em></small>.
+    </StarRating>
```
Comment (drewdrewthis) [inline src/molecules/StarRating/Readme.md]: @Jexeones24 Please add an example here of a `StarRating` component with its props so the next dev will know how it works.

File: src/organisms/cards/CompanyCard/Readme.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  </Layout>
+
+```
+
+
+
+
+
+
+
```
Comment (drewdrewthis) [inline src/organisms/cards/CompanyCard/Readme.md]: You can also probably delete these extra lines.

File: src/molecules/StarRating/__tests__/star_rating.spec.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+describe('<StarRating />', () => {
+
+  it('renders correctly', () => {
+    const wrapper = mount(<StarRating />);
+
+    expect(wrapper.type()).toEqual(StarRating);
+  });
+
+  it('renders correctly', () => {
+    const actual = renderer.create(<StarRating />).toJSON();
```
Comment (drewdrewthis) [inline src/molecules/StarRating/__tests__/star_rating.spec.js]: It's not essential for right now, and maybe leave this for another pull request, but it would be good to have these tests include the props that you'd expect.
